docs/source/error_code_list.rst

```diff
@@ -888,7 +888,7 @@ This produces the following output from mypy:
 .. code-block:: text
 
    example.py:3: error: "bool" is invalid as return type for "__exit__" that always returns False
-   example.py:3: note: Use "typing_extensions.Literal[False]" as the return type or change it to
+   example.py:3: note: Use "typing.Literal[False]" as the return type or change it to
```

Comment:
We should update the error message mypy emits here, but until we do so, we shouldn't change the docs here: this copies exactly the error message mypy emits on the above snippet https://mypy-play.net/?mypy=latest&python=3.12&gist=aab6f1cc25003dd1573cb3d8b8df2396